src/smc-webapp/project_page.cjsx

```diff
@@ -187,16 +190,43 @@ FreeProjectWarning = rclass ({name}) ->
         </div>
 
     render : ->
+        if @props.project
```

Comment:
Refresh your browser on a project -- this `@props.project` isn't defined until you move to another page and move back, so the warning banner doesn't appear.